core/src/main/java/org/jboss/galleon/universe/galleon1/LegacyGalleon1FeaturePackCreator.java

```diff
@@ -48,6 +53,32 @@ public void install(Universe<?> universe, FeaturePackLocation.FPID fpid, Path fp
         if(!(mvnUni.artifactResolver instanceof ArtifactRepositoryManager)) {
             throw new ProvisioningException(mvnUni.artifactResolver.getClass().getName() + " is not an instance of " + ArtifactRepositoryManager.class.getName());
         }
-        ((ArtifactRepositoryManager)mvnUni.artifactResolver).install(LegacyGalleon1Universe.toArtifactCoords(fpid.getLocation()), fpContentDir);
+
+        ArtifactCoords coords = LegacyGalleon1Universe.toArtifactCoords(fpid.getLocation());
+
+        Path tmpDir = null;
+        try {
+            tmpDir = Files.createTempDirectory("gln-mvn-feature-pack-creator");
```

Comment:
Could we simply create a temporary file? FeaturePackRepositoryManager will copy it with the correct name anyway.

Reply:
@aloubyansky done, as we commented by hipchat